packages/flutter_sprites/lib/src/action.dart

```diff
@@ -34,14 +39,18 @@ abstract class Action {
     _finished = false;
   }
 
+  /// The duration of this action in seconds.
```

Comment:
Try to avoid using the words used in the definition (in this case "duration" and "Action") in the documentation. We want to avoid writing docs that could be also written by someone who just saw the code for the first time and didn't know what anything actually did. :-)

For example here this should answer questions like what the timeline is (i.e. when does the duration start and end, what other things is it synchronised with).